### PR TITLE
Update the docstring in envs/toy_text/blackjack.py

### PR DESCRIPTION
**Pull request by sorryformyself**
In "Example 5.1 in Reinforcement Learning: An Introduction by Sutton and Barto", there says

`One of the dealer’s cards is face
up and the other is face down`

which doesn't even mention the player. 

In today's blackjack games, the player have two face up card in virtually all of those games. So it's better to make it clear here to avoid confusion. That's why I change a little bit docstring here.

### CODE REVIEW

File: gym/envs/toy_text/blackjack.py [hunk 1]
```diff
@@ -48,8 +48,8 @@ class BlackjackEnv(gym.Env):
     Face cards (Jack, Queen, King) have point value 10.
     Aces can either count as 11 or 1, and it's called 'usable' at 11.
     This game is placed with an infinite deck (or with replacement).
-    The game starts with each (player and dealer) having one face up and one
-    face down card.
+    The game starts with dealer having one face up and one face down card, while
+    player having two face up card. (Virtually for all Blackjack games today).
```
Comment (pzhokhov) [inline gym/envs/toy_text/blackjack.py]: typo `card` -> `cards`?

**Comment by pzhokhov** [comment]
Makes sense, dealer's strategy does not change based on player's cards, so player may as well have two open cards at the beginning. LGTM (other than the typo)

File: gym/envs/toy_text/blackjack.py (hunk 1 above)
Comment (sorryformyself) [reply gym/envs/toy_text/blackjack.py]: yeah that's just a typo, sorry for my mistake.

Comment (sorryformyself) [reply gym/envs/toy_text/blackjack.py]: > typo `card` -> `cards`?

yeah that's just a typo, sorry for my mistake.

**Pull request merged**